it/uk/gov/hmrc/homeofficesettledstatusstubs/endpoints/MrzSearchEndpointISpec.scala

```diff
@@ -99,7 +99,7 @@ class MrzSearchEndpointISpec extends ServerBaseISpec with JsonMatchers {
 
     "respond with 429 if status is 429" in {
       val result = publicFundsByMRZ(
-        s"""{"documentType":"NAT", "documentNumber" : "E8HDYKTB4", "nationality": "x", "dateOfBirth":"1956-10-09", $statusRange}""")
+        s"""{"documentType":"NAT", "documentNumber" : "E8HDYKTB5", "nationality": "x", "dateOfBirth":"1956-10-09", $statusRange}""")
```

Comment:
wait, is this not for internal server error? does this not fail?

Reply:
```
[info] - should respond with 429 if status is 429 *** FAILED ***
[info]   500 was not equal to 429 (MrzSearchEndpointISpec.scala:104)
```
Yep, this line just wants reverting :)

Reply:
sorted